helm/templates/logger-network-policy.yaml

```diff
@@ -0,0 +1,34 @@
+{{ if .Values.minio.metricsLogger.enable }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: minio-logger
+spec:
+  podSelector:
+    matchLabels:
+      app: minio-logger
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 0.0.0.0/0
```

Comment:
this allows any ip access to the logger, it should be limited to only the minio pod

Reply:
The [lines](https://github.com/CloudWebManage/cwm-worker-deployment-minio/pull/18/files#diff-29ac5e2135050d7babee37429aef8e4be08961983bdebecb4add10925003f08bR17-R19) below specify that only the `minio` pod is allowed.

Reply:
I think the from items are each handled separately, so if you specify ipBlock cidr 0.0.0.0/0 the other rules are meaningless